include/d/col/cc/d_cc_d.h

```diff
@@ -83,6 +83,9 @@ class dCcD_Unk : public cCcD_Obj, public cCcD_UnkAttr {
 
     virtual cCcD_ShapeAttr *GetShapeAttr() override;
 
+    // Have to bring these in scope. Consider renaming this
+    // Set function instead or create an inline?
+    using cM3dGUnk::Set;
```

Comment:
Using inlines don’t work, cM3dGLin::Set didn’t like this at least. 
Keep with `using`, `.cM3dG::Set`, or we could potentially rename something?